src/Extension/ExtensionServiceProvider.php

```diff
@@ -21,11 +21,10 @@ public function register()
     {
         $this->app->bind('flarum.extensions', 'Flarum\Extension\ExtensionManager');
 
-        $config = $this->app->make('flarum.settings')->get('extensions_enabled');
-        $extensions = json_decode($config, true);
+        $extensions = $this->app->make('flarum.extensions')->getEnabledExtensions();
 
         foreach ($extensions as $extension) {
-            if (file_exists($file = public_path().'/extensions/'.$extension.'/bootstrap.php')) {
+            if (file_exists($file = $extension->getPath().'/bootstrap.php')) {
```

Comment:
Can we shorten this to a method in the extension manager that returns an array of files to be included?